_dockerize/db-container/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mongo:bionic
+
+VOLUME ["/data/db"]
+WORKDIR /data
+
+# docker volume create mongo-data
+# docker run --rm -d --network vdh-network --name vdh-db -v mongo-data:/data/db db-container
+
+# use vdhObservationCollection
+# db.vdhObservationCollection.insertOne({id: "test"})
+# db.getCollection("vdhObservationCollection").find()
```

Comment:
This is not required. Since this Dockerfile is only adding a volume to the base image, it can be done when starting the container (like line 7 with `-v mongo-data:/data/db`). Plus, the logic of setting the proper workdir is probably manage in the base image.

Reply:
Thx I'll remove theses lines in my next commit